src/am_map.c

```diff
@@ -380,17 +380,8 @@ static void AM_addMark(void)
                             markpointnum_max*2 : 16) * sizeof(*markpoints),
                            PU_STATIC, 0);
 
-  // [crispy] keep the map static if not following the player
-  if (!followplayer)
-  {
-    markpoints[markpointnum].x = plr->mo->x >> FRACTOMAPBITS;
-    markpoints[markpointnum].y = plr->mo->y >> FRACTOMAPBITS;
-  }
-  else
-  {
```

Comment:
But this will always mark the center of the map, which is usually not what you want in non-follow mode. Here, you usually want to mark the player's place.

Reply:
Why? The mark in the center of the screen is vanilla behavior, it's what I expected. Though I've never seriously used marks.

Reply:
But in non-follow mode, I usually move the automap, look around and maybe mark something (I never actually did this). OK, if no one complains so far, maybe people use it differently.

Reply:
Can we open this back up? Why does Woof not behave consistently with vanilla or other source ports? If turn off follow mode, I expect to be able move the view to a place of interest like a secret or exit and mark that spot. It doesn't make any sense to have the marker placed at the player's location. If I wanted to do that, I would turn follow mode back on.

Here's an example. If I want to mark the rocket launcher in vanilla, I do it like this:

https://github.com/fabiangreffrath/woof/assets/56656010/05d908a4-f9b1-4667-988d-6ae1262f0f72

I can't do that in Woof:

https://github.com/fabiangreffrath/woof/assets/56656010/9bc05f4b-5433-4d8f-badf-766a39397d9d